Sources/OktaIdx/IDXResponse.swift

```diff
@@ -97,6 +97,7 @@ public class Response: NSObject {
                 switch result {
                 case .success(let token):
                     self.flow.send(response: token.result, completion: completion)
+                    self.flow.reset()
```

Comment:
Why is this reset needed? Also, could this potentially double-reset since there's a reset call in InteractionCodeFlow as well?

Reply:
This was an omission since it should always have performed a reset in the past, once authentication finishes.

As for the double reset, that can't happen, since there's two possible ways to exchange a token:
1. Through a successful response (e.g. the standard flow)
2. Through a redirect into the application, typically through a Social IDP redirect.

The reset is being called on a successful token exchange in either of those two places.